### PR TITLE
Use camel case instead of snake case

### PR DESCRIPTION
**Pull request by wangchenglong01**
#### What type of this PR
- Code Style

Use camel case instead of snake case. Keep code unification.

/flake

#### Which issue(s) this PR fixes:
Fixes #(issue-number)

### CODE REVIEW

**Comment by zhangzujian** [comment]
Thanks! Please squash the commits.

File: pkg/controller/vpc_nat_gateway.go [hunk 1]
```diff
@@ -33,14 +33,14 @@ var (
...
-	NAT_GW_SUBNET_ROUTE_ADD = "subnet-route-add"
-	NAT_GW_SUBNET_ROUTE_DEL = "subnet-route-del"
+	NatGwInit           = "init"
+	NatGwFloatingIpSync = "floating-ip-sync"
+	NatGwEipAdd         = "eip-add"
+	NatGwEipDel         = "eip-del"
+	NatGwSnatSync       = "snat-sync"
+	NatGwDnatSync       = "dnat-sync"
+	NatGwSubnetRouteAdd = "subnet-route-add"
+	NatGwSubnetRouteDel = "subnet-route-del"
```
Comment (zhangzujian) [inline pkg/controller/vpc_nat_gateway.go:43]: We should also make the first characters lowercase.

**Comment by wangchenglong01** [comment]
> Thanks! Please squash the commits.

The latest code has been commited.The commits have been squashed.

File: pkg/controller/vpc_nat_gateway.go (hunk 1 above)
Comment (wangchenglong01) [reply pkg/controller/vpc_nat_gateway.go:43]: The latest code has been commited. First characters have been changed lowercase.

Comment (wangchenglong01) [reply pkg/controller/vpc_nat_gateway.go:43]: This commit have been confused. I will make a new commit.

**Pull request closed without merging**